apps/files/src/components/SidebarQuota.vue

```diff
@@ -131,46 +129,47 @@ export default {
 					throw new Error('Invalid storage stats')
 				}
 
-				// Warn the user if the available account storage changed from > 0 to 0 
-				// (unless only because quota was intentionally set to 0 by admin in the interim)
-				if (this.storageStats?.free > 0 && response.data.data?.free <= 0 && response.data.data?.quota > 0) {
-					this.showStorageFullWarning()
-				}
-
 				this.storageStats = response.data.data
 			} catch (error) {
-				logger.error('Could not refresh storage stats', { error })
-				// Only show to the user if it was manually triggered
-				if (event) {
-					showError(t('files', 'Could not refresh storage stats'))
-				}
+				throw error
 			} finally {
 				this.loadingStorageStats = false
 			}
 		},
 
-		showStorageFullWarning() {
-			showError(this.t('files', 'Your storage is full, files can not be updated or synced anymore!'))
-		},
-
 		t: translate,
 	},
 }
 </script>
 
 <style lang="scss" scoped>
```

Comment:
thought(optional): large hover background

We might consider removing the lighter hover color for this button, because it looks rather unintentionally and oversized

Reply:
✅